tests/tools/test_converters.py

```diff
@@ -41,6 +43,41 @@ def test_polygon_data_file(self):
         assert np.isclose(new_quantity_block.value, 0.0)
 
 
+class TestConvertParameters:
+    def test_sample_data_file(self):
+        forcing = ExtOldForcing(
+            quantity=ExtOldQuantity.FrictionCoefficient,
+            filename="iniwaterlevel.xyz",
+            filetype=7,  # "Polyline"
+            method="5",  # "Interpolate space",
+            operand="O",
+        )
+
+        new_quantity_block = ParametersConverter().convert(forcing)
+        assert isinstance(new_quantity_block, ParameterField)
+        assert new_quantity_block.datafiletype == "sample"
+        assert new_quantity_block.interpolationmethod == "triangulation"
+
+
+class TestConvertMeteo:
+    def test_default(self):
+        forcing = ExtOldForcing(
+            quantity=ExtOldQuantity.WindX,
+            filename="windtest.amu",
+            filetype=4,
+            method="2",
+            operand="O",
+        )
+
+        new_quantity_block = MeteoConverter().convert(forcing)
+        assert isinstance(new_quantity_block, Meteo)
+        assert new_quantity_block.quantity == "windx"
+        assert new_quantity_block.operand == "O"
+        assert new_quantity_block.forcingfile == DiskOnlyFileModel("windtest.amu")
+        assert new_quantity_block.forcingfiletype == "meteoGridEqui"
+        assert new_quantity_block.interpolationmethod == "linearSpaceTime"
```

Comment:
Would you not prefer:
```suggestion
        assert new_quantity_block.interpolationmethod == MeteoInterpolationMethod.linearSpaceTime        
```
and similar for all other enum values in the above assertions?